fontforge/python.c

```diff
@@ -17105,16 +17119,16 @@ return( PySC_From_SC_I( sc ));
 static PyObject *PyFFFont_CreateInterpolatedGlyph(PyFF_Font *self, PyObject *args) {
     FontViewBase *fv;
     SplineFont *sf;
-    PyObject *from, *to;
+    PyObject *from, *to, *stricto=NULL;
```

Comment:
The layer code seems complete to me. This function calls `SplineCharInterpolate()` which calls `LayerInterpolate()` which calls `SplineSetsInterpolate()` *and* `InterpRefs()`. A complete compatibility check also needs to verify that the references are interpolation-compatible.